hr_contract_multi_jobs/views/hr_contract_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <!-- Inherited Contract view -->
+        <record id="hr_contract_view_form" model="ir.ui.view">
+            <field name="name">hr.contract.view.form</field>
+            <field name="model">hr.contract</field>
+            <field name="inherit_id" ref="hr_contract.hr_contract_view_form"/>
+            <field name="arch" type="xml">
+
+                <field name="job_id" position="replace">
+                    <field name="job_id" readonly="1"/>
```

Comment:
Please, if you only wish to add a readonly attribute to this element, use XPath attributes functionality instead than remove. It should be something like:

```
<field name="job_id" position="attributes">
    <attribute name='readonly'>1</attribute>
</field>
```